pdns/dnsdist.cc

```diff
@@ -2264,6 +2239,10 @@ try
     g_ACL.setState(acl);
   }
 
+  auto consoleACL = g_consoleACL.getCopy();
+  consoleACL.addMask("127.0.0.1/8");
```

Comment:
For completeness-sake, maybe we should add `::1/128` here as well?

Reply:
yes please, various things default to ::1 now...

Reply:
Makes sense!